packages/protocol/contracts/L1/TaikoL1.sol

```diff
@@ -287,4 +287,8 @@ contract TaikoL1 is EssentialContract, IHeaderSync, TaikoEvents {
     function getConfig() public pure virtual returns (TaikoData.Config memory) {
         return LibSharedConfig.getConfig();
     }
+
+    function getUncleProofDelay(uint256 blockId) public view returns (uint64) {
```

Comment:
please run `pnpm lint:sol`, you'll find the order of functions is incorrect: pure functions go after view functions.

Reply:
It looks like the renaming of the package.json from the tokenomics PR broke the husky pre-commit hooks.Will lint and push up.